### PR TITLE
[AMORO-2010] Modify Docker Image version to 'latest' in the startup script

### LINKED ISSUE #2010: [Improvement]: The version of the Amoro in the Docker startup script should be the latest by default
### Search before asking

- [X] I have searched in the [issues](https://github.com/NetEase/amoro/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

When using docker with the AMS. We should download a script named `demo-cluster.sh` and start with the version.
And the default version in the script is the latest release version.
It's not convenient for release because we should change the doc and the script every time we release.

### How should we improve?

The version of the Amoro in the Docker startup script should be the latest by default so we don't need to change the version in the script and the doc every time we release a new version.
The docker image covered both the latest version and the release version.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Subtasks

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by HuangFru**
## Why are the changes needed?

Close #2010.

## Brief change log

- Change 'PROJECT_VERSION' to 'latest'.
- Remove useless alias

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

File: docker/demo-cluster.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #
 
 
-PROJECT_VERSION=0.5.0-SNAPSHOT
+PROJECT_VERSION=latest
```
[the remark this answers is not in the document]
Comment (HuangFru) [reply docker/demo-cluster.sh:21]: Docker Image has the default value 'latest', we overwrite the 'latest' version of the docker image every time we release it.

**Review by wangtaohz — APPROVED** [review]
LGTM
